katello.dot

```diff
@@ -2,108 +2,120 @@
 
 [Browser] as browser
 
-node "server" {
-  [Postgres] as postgres
-  [Candlepin] as candlepin
+node "Main Environment" {
+	
+	node "Server" {
+	  [Postgres] as postgres
+	  
 
-  [Elasticsearch] as elasticsearch
-  [MongoDB] as mongo_server
-  [Qpid] as qpidd_server
-  [Dispatch Router] as qdrouterd_server
-  [Smart Proxy] as server_smart_proxy
+	  [MongoDB] as mongo_server
+	  [Qpid] as qpidd_server
+	  [Dispatch Router] as qdrouterd_server
+	  [Smart Proxy] as server_smart_proxy
+	  
+	  node "Tomcat" as tomcat_server {
+		  [Candlepin] as candlepin
+	  }
 
-  node "Apache" as server_apache {
+	  node "Apache" as server_apache {
 
-    node "Pulp" as pulp_server {
-      [API] as pulp_server_api
-      [pulp_* processes] as pulp_server_backend
-    }
+		node "Pulp" as pulp_server {
+		  [API] as pulp_server_api
+		  [pulp_* processes] as pulp_server_backend
+		}
 
-    [Puppet Master] as puppet_master
+		[Puppet Master] as puppet_master
 
-    node "Foreman Application" {
-      [Foreman] as foreman
-      [Katello] as katello
-      [Foreman Tasks] as foreman_tasks
-      [Dynflow] as dynflow
-      [Foreman Docker] as foreman_docker
-      [Foreman Discovery] as foreman_discovery
-      [Foreman SCAP] as foreman_scap
+		node "Foreman Application" {
+		  [Foreman] as foreman
+		  [Katello] as katello
+		  [Foreman Tasks] as foreman_tasks
+		  [Dynflow] as dynflow
+		  [Foreman Docker] as foreman_docker
+		  [Foreman Discovery] as foreman_discovery
+		  [Foreman SCAP] as foreman_scap
```

Comment:
Feel fre eif you want to add things like:

 * Foreman Templates
 * Foreman Ansible
 * Foreman Virt Who Configure